tests/run_tests.sh

```diff
@@ -50,7 +50,14 @@ function run {
             make -f Makefile.$BUILDBOT_PLATFORM run-$name
         fi
     done
+<<<<<<< HEAD
```

Comment:
Merge conflict left in. Please force push an update.

Reply:
🤦

Reply:
Fixed in force-pushed [f509753](https://github.com/capablevms/cheri-examples/pull/92/commits/f509753a0b6fa89ee15be5499c747ccb018fb704).